src/mon/PGMonitor.cc

```diff
@@ -1473,6 +1476,16 @@ void PGMonitor::dump_pool_stats(stringstream &ss, Formatter *f, bool verbose)
           << pool_id;
       if (verbose)
 	tbl << "-";
+
+      if (pool->quota_max_objects == 0)
+        tbl << "N/A";
+      else
+        tbl << si_t(pool->quota_max_objects);
+         
```

Comment:
@songbaisen probably the spaces are here. and you can use
```
git show --color
```
to see the trailing spaces.